test/negative_tests/hang.result

```diff
@@ -0,0 +1,25 @@
+-- test-run result file version 2
+--  box.schema.user.grant('guest', 'replication')replication
+local test_run = require('test_run').new()
```

Comment:
Commands here are different from the test itself. Anyway, is there any sense in placing a result file for this test? We expect a failure before comparing output with it.

Reply:
Not relevant anymore.